boards/arm/mimxrt1064_evk/flash_flexspi_nor_lut.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2021 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "flash_mcux_flexspi_nor_lut_index.h"
+
+static const uint32_t flash_flexspi_nor_lut[][4] = {
```

Comment:
I don't like defining the LUT at the board level because it will have to be duplicated for every other board that uses the same flash. I'd prefer instead to keep the LUT defined in `drivers/flash` because that allows reuse by both in-tree and out-of-tree boards. I think we should either 1) conditionalize the LUT in `drivers/flash/flash_mcux_flexspi_nor.c`, or 2) introduce a new `drivers/flash/flash_mcux_flexspi_ospi.c` or `drivers/flash/flash_mcux_flexspi_mx25um51345g.c`. I'm leaning toward option 2.

Reply:
So `drivers/flash/flash_mcux_flexspi_mx25um51345g.c` would be required since LUT is memory specific. If not duplicating LUT header, is duplicating source code `flash_mcux_<memoryName>.c`  acceptable for planned device support in future?

Reply:
Yes

Reply:
Ok, will do. Please take  a note that devices to be supported in future with same memory model may not be able to use this LUT. Because LUT, specifically dummy cycles used for READ sequence would change based on frequency at which flash is operated. Also, this LUT is valid only for Octal mode but in case of SPI/DPI/QPI modes with same memory model would require an update to LUT.

Reply:
Done, moved LUT to driver.